integration_tests/sdk/delete_workflow_test.py

```diff
@@ -159,6 +167,12 @@ def test_delete_workflow_saved_objects_twice(client):
             client.delete_flow(flow_2_id, saved_objects_to_delete=tables, force=True)
         assert str(e_info.value).startswith("Failed to delete")
 
+        # Failed to delete tables, but flow should be removed.
+        with pytest.raises(Exception) as e_info:
+            client.delete_flow(flow_2_id)
+        assert str(e_info.value).startswith("The organization does not own")
+        flow_ids_to_delete.remove(flow_2_id)
```

Comment:
maybe instead of this no-op, maybe can assert that `len(flow_ids_to_delete) == 0`? A `remove()` call that doesn't do anything is just another source of confusion.

Reply:
I think this comment is related to the previous one. I actually created two flows so this isn't a no-op.

Reply:
Can you verify that the flow is indeed deleted? Maybe do a client.flow() and check that an InvalidUserArgumentException occurs.